scripts/fix-badge-links.js

```
@@ -9,31 +9,31 @@
+    name: "Trailing backticks",
     regex: /(https?:\/\/[^`\s]+)`(?=[\s\n]|$)/g,
-    replacement: '$1',
+    replacement: "$1",
   },
   {
-    name: 'Incomplete workflow badge URLs (branch param)',
-    regex: /(https?:\/\/github\.com\/[^\/]+\/[^\/]+\/actions\/workflows\/[^\s?]+\.yml)\/badge\.svg\?branch=([^\s&)]+)$/gm,
-    replacement: '$1/badge.svg?branch=$2',
+    name: "Incomplete workflow badge URLs (branch param)",
+    regex:
+      /(https?:\/\/github\.com\/[^\/]+\/[^\/]+\/actions\/workflows\/[^\s?]+\.yml)\/badge\.svg\?branch=([^\s&)]+)$/gm,
```

ESLint

[error] Unnecessary escape character: \/.

(no-useless-escape)